Assets/Scripts/LookedAtInteractiveDisplayText.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// This UI text displays info about the currently looked at interactive IInteractive.
+/// The text should be hidden if the player is not currently looking at an interactive element.
+/// </summary>
+
+public class LookedAtInteractiveDisplayText : MonoBehaviour
+{
+    private IInteractive lookedAtInteractive;
+    private Text displayText;
+
+    private void Awake()
+    {
+        displayText = GetComponent<Text>();
+    }
+
+    private void UpdateDisplayText()
+    {
+        if (lookedAtInteractive != null)
+        {
+            displayText.text = lookedAtInteractive.DisplayText;
+        }
+        else
+        {
+            displayText.text = string.Empty;
+        }          
+    }
+
+    /// <summary>
+    /// Event handler for DetectLookedAtInteractive.LookedAtInteractiveChanged
+    /// </summary>
+    /// <param name="newLookedAtInteractive">Reference to the new IInteractive the player is looking at.</param>
+    private void OnLookedAtInteractiveChanged(IInteractive newLookedAtInteractive)
+    {
+        lookedAtInteractive = newLookedAtInteractive;
+        UpdateDisplayText();
+    }
+
+    #region Event subscription / unsubscription
+    private void OnEnable()
+    {
+        DetectLookedAtInteractive.LookedAtInteractiveChanged += OnLookedAtInteractiveChanged;
+    }
+    private void OnDisable()
+    {
+        DetectLookedAtInteractive.LookedAtInteractiveChanged -= OnLookedAtInteractiveChanged;
+
```

Comment:
Remove extra lines like this unless they are consistently used in a way that enhances readability.